### DISCUSSION TITLE
Do deepocsort and botsort only use the reid model?

### DISCUSSION THREAD

**Discussion by xiyuskang** [mikel-brostrom/yolo_tracking#969] [Q&A]
How do I use the custom yolov8 model to create the track class? Do deepocsort and botsort only use the reid model?

**Comment by mikel-brostrom** [comment]
> How do I use the custom yolov8 model to create the track class?

https://github.com/mikel-brostrom/yolo_tracking#custom-object-detection-model-example

> Do deepocsort and botsort only use the reid model?

DeepOCSORT, BoTSORT and StrongSORT use both motion and appearance information in the tracking process. Motion information is generated by kalmanfilters while the appearance information is generated by a ReID model.

**Reply by xiyuskang** [reply]
How can I use these track classes in python by using my own yolov8 model?
